qa/sc_evm_sc_fork.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import logging
+
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from SidechainTestFramework.scutil import generate_next_block
+from test_framework.util import assert_equal, assert_false
+
+
+class SCEVMSCFork(AccountChainSetup):
```

Comment:
this one is testing GasFeeFork, can you please rename it, as the name seems too generic? Also, can you add a couple of valid/invalid transactions for the gas limit condition, f.e. invalid with 22000000 at epoch 4, valid with 25000000 at epoch 5

Reply:
done